NEWS.md

```diff
@@ -1,3 +1,24 @@
+## 19.3.0 2020-07-28
+
+* Add 'Aged to lost' status to allowed item statuses list (MODINVSTOR-503)
+* Make migratePrecedingSucceedingTitles.sql idempotent (MODINVSTOR-541)
```

Comment:
What impact does this have for an organisation choosing to upgrade to this version?

Reply:
If upgrading to 19.2.4 fails with "duplicate key violates unique constraint »preceding_succeeding_title_pkey«" it will not fail when upgrading to 19.3.0 because this bug has been fixed.

Reply:
I looked back through all the merges of mainline starting from previous feature release whichg was 19.2.0 and picked featires had fixed. This partly intersect with list of changes for release: https://issues.folio.org/browse/MODINVSTOR-541?jql=project%20%3D%20MODINVSTOR%20AND%20fixVersion%20%3D%2019.3.0. But thewe were sme featiures not listed in that list:
MODINVSTOR-498: Expand oai-pmh view with additional fields
MODINVSTOR-524: Change permissions location for oai-pmh view
MODINVSTOR-499: Add Index for Instances Full-text Subjects Search

So it looks like here in release notes we have whole list of changes statring from previous feature release.

Reply:
@julianladisch could that be reworded as something like `upgrade preceding / succeeding titles is now idempotent`?

Reply:
It looks like here are placed names form Jira tickets. Do you mean thta some names needs to be chnages to have more sence as for release notes?

Reply:
That is my personal preference. I think of the audience of the release notes as folks who are considering (or have) upgraded to this version of the module and want to understand, from a behavioural or operational perspective what has changed.

It would be great if that could be done directly from the names of issues in JIRA or the commit log in git, however I've found that not to be the case on FOLIO.

Reply:
I reworded title of this feature. Could ypu please check if name is cirrently became more undestandable.

Reply:
Several times when preparing release notes/news I have changed the Jira summary to be more understandable and then copied the new summary into the release notes/news.